src/parser/jobs/drk/modules/OGCDDowntime.ts

```diff
@@ -11,12 +11,15 @@ export class OGCDDowntime extends CooldownDowntime {
 		},
 		{
 			cooldowns: [this.data.actions.DELIRIUM],
-			firstUseOffset: 15000,
+			firstUseOffset: 5000,
```

Comment:
While English openers use Delirium early and sit on the stacks, Japanese don't, so I'd leave this one alone or set it to like 7500ms (JP seems to do it after 3rd GCD).

Reply:
Updated.